src/plugins/cordova-plugin-device-motion/sim-host.css

```diff
@@ -6,8 +6,10 @@
     margin-right: auto;
 }
 
-#accelerometer-canvas {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-}
\ No newline at end of file
+#device-motion #accelerometer-canvas {
+    margin: 0 auto;
+    position: relative;
+
+    /* Weird percentage is 200/294, which keeps size the same as it used to be */
```

Comment:
Where does 294 come from? It looks like the previous code was using `200/320` as the scaling factor